fluent-bundle/src/memoizer.ts

```diff
@@ -0,0 +1,14 @@
+type IntlCache = WeakMap<object, Record<string, object>>;
```

Comment:
Just a question: why is better to use a `WeakMap` here instead of just a plain object?

Reply:
I don't know :) Kept it as it is right now.
Seeing that map keys are `Intl.NumberFormat` and `Intl.DateTimeFormat` and will not be garbage collected it should be regular `Map` instead

Reply:
That's a good point. @zbraniecki, do you remember why we went with a `WeakMap` here?

Reply:
I didn't fully finish my doodling, but it seems that the keys are objects and not strings?

Reply:
I now added commit that switches from `WeakMap` to `Map` as keys that are used here cannot be garbage collected anyway.